### PR TITLE
Fixes up alerts

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Gets rid of old alerts, fixes up some bugs relating to alerts, doctored up sprites to actually work and look clean, redid some of the backend shit to work better, also changed text to fit SCP better. Code PITCHblack (hostile GoI onboard) is now selectable, there are descriptions of alert levels in the command menu, probably some other crap that I'm forgetting, seriously this thing is HUGE

## Why It's Good For The Game

Less immersion-breaking text, less bugs, better sprites, less old code the better

## Changelog

:cl:
add: You can now actually select the second-to-last alert (Code Pitchblack)!
add: Command program now has descriptions of all alerts!
del: All old alert levels are gone (Except Delta)
fix: Alerts now have more sensical description texts
imageadd: Fixed up alert sprites to work with other sprites
imagedel: Deleted old alert sprites
refactor: Messy alert code was standardized, less dumb shit
refactor: In general, messy code relating to alerts has been cleaned up
/:cl:

### CODE REVIEW

File: code/game/response_team.dm [hunk 1]
```diff
@@ -99,7 +99,7 @@ var/can_call_ert
 	while(send_emergency_team == 0) // There is no ERT at the time.
 		var/decl/security_state/security_state = decls_repository.get_decl(GLOB.using_map.security_state)
 		var/index = list_find(security_state.all_security_levels, security_state.current_security_level)
-		ert_base_chance += 2**index
+		ert_base_chance += index
```
Comment (MACIEKBAKI) [inline code/game/response_team.dm:102]: Why the change 'ere?

File: code/datums/security_state.dm [hunk 2]
```diff
@@ -208,53 +173,146 @@
...
+
+/decl/security_level/default/code_black/switching_up_to()
+	security_announcement_black.Announce("The site is experiencing multiple Keter and Euclid level containment breaches. Full site lockdown initiated.", "Attention! Code Black alert procedures now in effect!")
+	notify_station()
+
+/decl/security_level/default/code_black/switching_down_to()
+	security_announcement_black.Announce("The Site has been secured from subversive elements. Security is to sweep the facility and recontain all dangerous SCPs immediately.", "Attention! Code Black alert procedures now in effect!")
+	notify_station()
+
+/decl/security_level/default/code_ultrablack
```
Comment (Vizoplez) [inline code/datums/security_state.dm]: This is goofy. Very goofy. Why would we use code ULTRABLACK instead of CODE GREY (which already exists) and serves a better purpose?

File: maps/site53/site53.dm [hunk 3]
```diff
@@ -14,7 +14,7 @@
 //	#include "torch_unit_testing.dm"
 //	#include "torch_antagonism.dm"
 //	#include "torch_npcs.dm"
-	#include "site53_securitystate.dm"
+//	#include "site53_securitystate.dm"
```
Comment (MACIEKBAKI) [inline maps/site53/site53.dm]: As you deleted the file, you could just remove this line entirely instead of commenting it out

File: code/datums/security_state.dm [hunk 4]
```diff
@@ -208,53 +173,146 @@
...
+	description = "A Keter SCP is currently uncontained."
+
+	var/static/datum/announcement/priority/security/security_announcement_red = new(do_log = 0, do_newscast = 1, new_sound = sound('sound/AI/announcer/codered.ogg'))
+
+/decl/security_level/default/code_red/switching_up_to()
+	security_announcement_red.Announce("A Keter SCP has broken containment and its current whereabouts are unknown. Security should secure all exit points immediately before recontaining breached anomalies.", "Attention! Code red alert procedures now in effect!")
+	notify_station()
+
+/decl/security_level/default/code_red/switching_down_to()
+	security_announcement_red.Announce("All major threats to the Site have been neutralized or contained, but one or more Keter SCPs remain at large. Security should focus their efforts on recontaining the SCP. Full site lockdown disengaged.", "Attention! Code red alert procedures now in effect!")
```
Comment (MACIEKBAKI) [inline code/datums/security_state.dm]:
```suggestion
	security_announcement_red.Announce("All major threats to the Site have been neutralized or contained, but one or more Keter SCPs remain uncontained. Security should focus their efforts on recontaining the escaped SCP. Full site lockdown disengaged.", "Attention! Code red alert procedures now in effect!")
```

but this also sounds kinda meh

File: code/datums/security_state.dm [hunk 5]
```diff
@@ -208,53 +173,146 @@
...
+	name = "code yellow"
 
 	light_max_bright = 0.5
-	light_inner_range = 0.1
+	light_inner_range = 1
 	light_outer_range = 2
-	light_color_alarm = COLOR_BLUE
-	light_color_status_display = COLOR_BLUE
+	light_color_alarm = COLOR_VIOLET
+	light_color_status_display = COLOR_VIOLET
```
Comment (ghost) [inline code/datums/security_state.dm]: @cheesePizza2 Wtf?

File: code/datums/security_state.dm (hunk 2 above)
Comment (ghost) [reply code/datums/security_state.dm]: Code grey is weaksauce, homechacho. 

Code ULTRABLACK is the yeez kneez.

File: code/datums/security_state.dm [hunk 6]
```diff
@@ -194,8 +194,8 @@
 	light_max_bright = 0.5
 	light_inner_range = 1
 	light_outer_range = 2
-	light_color_alarm = COLOR_VIOLET
-	light_color_status_display = COLOR_VIOLET
+	light_color_alarm = COLOR_YELLOW
+	light_color_status_display = COLOR_YELLOW
```
Comment (ghost) [inline code/datums/security_state.dm]: Wtf?

File: code/datums/security_state.dm (hunk 2 above)
Comment (aidan-bowman) [reply code/datums/security_state.dm]: changed to pitchblack (recommended by lore guy 2)

File: code/game/response_team.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/game/response_team.dm:102]: This was designed for 3 usable alert levels, but we have 6

File: code/datums/security_state.dm (hunk 5 above)
Comment (aidan-bowman) [reply code/datums/security_state.dm]: changed to the PROPER color_yellow

File: code/datums/security_state.dm (hunk 6 above)
Comment (aidan-bowman) [reply code/datums/security_state.dm]: fuck you, thats why

File: maps/site53/site53.dm (hunk 3 above)
Comment (aidan-bowman) [reply maps/site53/site53.dm]: done

File: code/datums/security_state.dm [hunk 7]
```diff
@@ -208,53 +173,146 @@
...
+	name = "code yellow"
 
 	light_max_bright = 0.5
-	light_inner_range = 0.1
+	light_inner_range = 1
 	light_outer_range = 2
-	light_color_alarm = COLOR_BLUE
-	light_color_status_display = COLOR_BLUE
+	light_color_alarm = COLOR_YELLOW
+	light_color_status_display = COLOR_YELLOW
```
Comment (CristoMagisto69) [inline code/datums/security_state.dm:198]: Wtf

File: code/game/response_team.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/game/response_team.dm:102]: For reference, if the facility is at Code Black for 30 minutes, that ups the base chance by 50%

File: code/datums/security_state.dm [hunk 8]
```diff
@@ -260,7 +260,7 @@
 	notify_station()
 
 /decl/security_level/default/code_red/switching_down_to()
-	security_announcement_red.Announce("All major threats to the Site have been neutralized or contained, but one or more Keter SCPs remain at large. Security should focus their efforts on recontaining the SCP. Full site lockdown disengaged.", "Attention! Code red alert procedures now in effect!")
+	security_announcement_red.Announce("All major threats to the Site have been neutralized or contained, but one or more Keter SCPs remain uncontained. Security should focus their efforts on recontaining the escaped SCP. Full site lockdown disengaged.", "Attention! Code red alert procedures now in effect!")
```
Comment (CristoMagisto69) [inline code/datums/security_state.dm:263]: Uncontained sucks. 

At large has more weight and conveys the fact they should be found and dealt with.

"Uncontained" is weak.

**Comment by aidan-bowman** [comment]
Why draft the PR? I know I was banned but I still want to see my work merged.

**Comment by Tsurupeta** [comment]
Because it had issues with LORE team, this had nothing to do with you being banned on discord.

**Comment by aidan-bowman** [comment]
> Because it had issues with LORE team, this had nothing to do with you being banned on discord.

damn sorry, the timing just threw me off. what issues specifically?

**Comment by 77seven77seven** [comment]
merge this shit already

**Comment by 77seven77seven** [comment]
happy birthday PR

**Comment by aidan-bowman** [comment]
Merge conflict should be resolved.

**Comment by Tsurupeta** [comment]
How does alarm panel emit black light????

**Comment by aidan-bowman** [comment]
> How does alarm panel emit black light????

Ultraviolet lights? I figured it would be confusing to have the alert colors different than the alert label (as the old system did), and I'd hardly think it'd have any effect on gameplay if it was "unrealistic" (considering SCP has moving statues and magic 1400s plague doctors).

**Comment by Tsurupeta** [comment]
It should be fine (I hope).
